launchdarkly/README.md

```diff
@@ -1,45 +1,56 @@
 # LaunchDarkly Integration
 
-## Overview
+### Overview
 
-This integration tracks metrics from [LaunchDarkly's relay proxy][1], such as number of proxied stream connections and proxied API route requests.
+Deploying features with LaunchDarkly is great, but it can be difficult to see
+the effects of these deployments on users' services or systems. Datadog's
+monitoring and analytics tools for modern DevOps teams allow their customers to
+observe the health of their systems over time. The LaunchDarkly integration with
+Datadog exposes LaunchDarkly events in context with events gathered by the
+Datadog agents. This makes it easier to reason about changes to customers'
+systems as a result of feature deployments. For example, if a feature gets
+deployed that causes a service to slow down, a DevOps engineer can see the cause
+within Datadog.
 
-## Setup
+![LaunchDarkly events in Datadog][2]
 
-### Configuration
+### Setup
 
-After you've configured the [relay proxy][2], add the following section to your LaunchDarkly relay instance's `ld-relay.conf` file:
+#### Configuration
 
-```
-[datadog]
-enabled=true
-statsAddr="YOUR_STATS_ADDRESS"
-```
+This integration will need a [Datadog API key][1]. Only Datadog admins can
+create an API key. Once you've obtained a Datadog API key, visit the
+[LaunchDarkly documentation for the Datadog integration][3] for further
+instructions on how to register the API key in LaunchDarkly.
 
-## Data Collected
+### Data Collected
 
-### Metrics
+The LaunchDarkly integration sends the following data to Datadog:
```

Comment:
Right now it only sends events though. Do you know if we could get a list of all metrics collected by this integration in oder to display them here ? And say something along the lines of

`Find below a list of metrics that are collected after [enabling XXX]()`

Reply:
This integration doesn't send any metrics to DD. I wasn't sure if it was a requirement to have a Metrics section. I just copied what was there from the prior readme.